extensions.md

```diff
@@ -186,3 +186,53 @@ the server, but LSP does not expose them.
   - Clients that don't support client-side filtering should ignore this and
     use `sortText`, which incorporates `score` and fuzzy-matching.
 
+## AST
+{:v12}
+
+C++ has a complicated grammar and semantic structure that's not always obvious
+from the source. Inspecting the clang AST can lend some insight.
```

Comment:
"clang" -> "Clang" (? I'm still confused when we do clangd but I guess Clang prefers capitalization)
"lend" -> "provide"/"give"? the idiom does not look familiar for me as a non-native speaker.

Reply:
(clang->Clang: done)

Yeah "lend some insight" is an idiom - but it seems like one where the meaning is clear even if it's unfamiliar, just seems like maybe a strange choice of words?

(I prefer "lend" over "provide" here as it's a bit more tentative/modest, and the AST still takes some deciphering)